book/src/drivers.md

```diff
@@ -80,6 +80,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Nutanix](https://github.com/nutanix/csi-plugin) | `com.nutanix.csi` | v0.3, v1.0, v1.2 | A Container Storage Interface (CSI) Driver for Nutanix | Persistent | "Read/Write Single Pod" with Nutanix Volumes and "Read/Write Multiple Pods" with Nutanix Files | Yes | Raw Block, Snapshot, Expansion, Cloning
 [OpenEBS](https://github.com/openebs/csi)| `cstor.csi.openebs.io` | v1.0 | A Container Storage Interface (CSI) Driver for  [OpenEBS](https://www.openebs.io/)| Persistent | Read/Write Single Pod | Yes | Expansion, Snapshot, Cloning
 [Open-E](https://github.com/open-e/JovianDSS-KubernetesCSI) | `com.open-e.joviandss.csi` | v1.0 | A Container Storage Interface (CSI) Driver for Open-E JovianDSS Storage  | Persistent | Read/Write Single Pod | Yes | Snapshot, Cloning 
+[Open-Local](https://github.com/oecp/open-local) | `csi.storage.oecp.io` | v0.1 | A Container Storage Interface (CSI) Driver for Local Storage  | Persistent | Read/Write Single Pod | Yes | Raw Block, Expansion, Snapshot
```

Comment:
Is v0.1 driver version?  This column is for CSI Spec version supported by the CSI driver.

Reply:
made an unchecked error, it's been updated.